browser/src/App.tsx

```diff
@@ -73,11 +73,8 @@ const Banner = styled.div`
 
 const BANNER_CONTENT = (
   <>
-    gnomAD v4.1.0 is now available. Read our {/* @ts-expect-error */}
-    <ExternalLink href="https://gnomad.broadinstitute.org/news/2024-04-gnomad-v4-1">
-      blog post
-    </ExternalLink>{' '}
-    for more details
+    Help us continue to improve gnomAD by taking 5 minutes to fill out our {/* @ts-expect-error */}
```

Comment:
one of these days I will fix GBTK so we don't have to include a `ts-expect-error` every time we use one of our own widgets